x/eibc/types/demand_order.go

```diff
@@ -32,6 +32,25 @@ func NewDemandOrder(rollappPacket commontypes.RollappPacket, price, fee math.Int
 }
 
 func (m *DemandOrder) ValidateBasic() error {
+	if len(m.Price) > 1 || len(m.Fee) > 1 {
+		return ErrMultipleDenoms
+	}
```

Comment:
I don't understand it
Please comment

Reply:
the `DemandOrder` struct have `sdk.Coins` field
the validation checks it's not contains multiple coins